netman/adapters/switches/juniper/base.py

```diff
@@ -215,15 +215,20 @@ def set_access_mode(self, interface_id):
         if len(update_attributes) > 0:
             update = Update()
             update.add_interface(interface_update(interface_id, "0", update_attributes))
-
-            self._push(update)
+            try:
```

Comment:
can you put this try except in a _push_interface_update that would always handle the interface not found error?